frontend/src/components/chat/ChannelView.css

```
@@ -555,6 +581,138 @@
+  font-size: var(--font-size-xs);
+  text-transform: lowercase;
+  cursor: pointer;
+}
+
+.ch-msg__confirm-btn--danger {
+  color: var(--status-error);
+}
+
+.ch-msg__confirm-btn:hover:not(:disabled),
+.ch-msg__confirm-btn:focus-visible:not(:disabled) {
+  border-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)